config/initializers/assets.rb

```diff
@@ -18,6 +18,7 @@
                   host_edit
                   host_edit_interfaces
                   hosts
+                  jquery.ui.custom_spinners
```

Comment:
I misread the line, I thought it was the generic ui spinner lib, ok makes sense, thanks